resources/puppetlabs/lein-ezbake/template/pe/ext/default.erb

```diff
@@ -13,7 +13,14 @@ USER="<%= EZBake::Config[:user] %>"
 GROUP="<%= EZBake::Config[:group] %>"
 INSTALL_DIR="/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>"
 CONFIG="/etc/puppetlabs/<%= EZBake::Config[:real_name] %>/conf.d"
+
+# Bootstrap path
+<% if EZBake::Config[:bootstrap_source] == 'services-d' -%>
+BOOTSTRAP_CONFIG="/etc/puppetlabs/<%= EZBake::Config[:real_name] %>/services.d/,${INSTALL_DIR}/config/services.d/"
```

Comment:
I'm sure this is legacy, but what's the point of config being the path name? `etc` kind of implies that.